src/main/java/io/federecio/dropwizard/swagger/SwaggerBundle.java

```diff
@@ -40,7 +41,7 @@
     public void initialize(Bootstrap<?> bootstrap) {
         bootstrap.addBundle(new ViewBundle<Configuration>() {
             @Override
-            public ImmutableMap<String, ImmutableMap<String, String>> getViewConfiguration(final Configuration configuration) {
+            public Map<String, Map<String, String>> getViewConfiguration(final Configuration configuration) {
```

Comment:
This was changed in https://github.com/dropwizard/dropwizard/commit/7c8705294e1e335848f0a1a23b62f54aa30b2ee7